### DISCUSSION TITLE
Activate Postgres extension

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [supabase/supabase#22369] [Questions]
I want to use Apache AGE(https://github.com/apache/age/tree/master) which is an extension for Postgres. I tried following the steps on the repo but it didn't get registered with Supabase.

**Comment by GaryAustin1** [comment]
If you are talking about a Supabase hosted instance you can't install your own extensions unless they happen to be simple pure SQL.

**Comment by Tejaswgupta** [comment]
@GaryAustin1 Yes we've self hosted a Supabase instance, is there any workaround for this?

**Reply by GaryAustin1** [reply]
My understanding with self hosted you can install other extensions.   BUT I don't do self-hosted and don't get to involved with helping with those issues beyond the basics.  So you probably need to be more specific on the issue you are having installing for others to help, or narrow down it is a Supabase issue versus just an issue installing the extension.

This is a user commenting on installing another extension https://github.com/orgs/supabase/discussions/2239#discussioncomment-6897458 on self-hosted.  Not sure it helps or not.
